packages/jetpack-ai-calypso/test/index.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * @jest-environment jsdom
+ */
+
+/**
+ * External dependencies
+ */
+import '@testing-library/jest-dom';
+import { screen } from '@testing-library/dom';
```

Comment:
Is there a good reason not to use `screen` from `@testing-library/react`? That way we'll only need to declare `@testing-library/react` as a dependency.